### ISSUE TITLE
Home Screen Personalization: QA feature signoff ticket

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#7517]
## Post-develop merge testing
After the personalization feature merges to develop, QA needs to complete these tasks before signoff:

- [x] Happy path testing for key feature functionality
- [ ] Unhappy path testing for key feature functionality
- [x] General app regression testing
- [x] Feature-specific regression testing
- [ ] Cases written & organized correctly in TestRail for future manual testing

### Bugs during implementation
_Running bug list to serve as inspiration/fodder for feature-specific regression testing. Specific test plan documented elsewhere in this ticket_
- App crash when module count of 0
- Claims navigation back out of claims list, when coming from home screen as user without claims decision letters access
- #7746
- Consistent "claims only", "appeals only", "claims and appeals" kerfuffles
- APIs called again when they need to be (on home screen refresh; in appropriate lists)
- APIs not called repeatedly when they DON'T need to be (on home screen initial load, HS refresh, various lists)
- Android for several components, announcing in pieces instead of a whole with SR
- Correct handling of null/empty states (home & category screens)
- Correct fonts
- Existence of drop shadows
- Cancel appointments day limit (esp with appt componentization moving around the same time)
- Hierarchy of subtext messages (error then null then other)

**Comment by TKDickson** [comment]
Pre-develop testing will be [this test run](https://dsvavsp.testrail.io/index.php?/runs/view/5653&group_by=cases:section_id&group_order=asc) (which is mostly looking for high-level regressions, and also covers a few of the specific bugs above)

Plus these one-off checks:
- [ ] App does not crash with module count of 0
- [ ] Claims navigation back out of claims list, when coming from home screen as user without claims decision letters access
- [ ] Hierarchy of subtext messages (error then null then other)

**Comment by TKDickson** [comment]
Have a draft PR for of the [QA findings for this feature ](https://github.com/department-of-veterans-affairs/va.gov-team/pull/87142)

**Comment by TKDickson** [comment]
Completed planning testing. Found one issue, not caused by HSP (also saw it in develop) - #8944.

Merge to develop is still blocked by detox tickets (marked as blockers on this ticket), and I need to track down to get final UX signoff from those folks (I've heard that Brea is done, but haven't heard from Holly or Lauren).

**Comment by TKDickson** [comment]
Pre-develop regression testing is complete; [QA findings page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/design-personalization/qa/findings.md) is live; UX team confirmed their pre-merge testing is complete; all detox scripts are passing with the HSP updates. Approved to merge into develop from the QA standpoint, as early as tomorrow. @dumathane @ajsarkar28

**Comment by TKDickson** [comment]
When Tom finishes RC testing for the previous round, I need to update these manual testing cases:
[home screen](https://dsvavsp.testrail.io/index.php?/cases/view/37243&group_by=cases:section_id&group_id=2164&group_order=asc&display_deleted_cases=0)
[veteran status](https://dsvavsp.testrail.io/index.php?/cases/view/48413)
[veteran status profile](https://dsvavsp.testrail.io/index.php?/cases/view/48414&group_by=cases:section_id&group_id=2163&group_order=asc&display_deleted_cases=0)
[benefits landing](https://dsvavsp.testrail.io/index.php?/cases/view/37245&group_by=cases:section_id&group_id=9688&group_order=asc&display_deleted_cases=0)
[health landing](https://dsvavsp.testrail.io/index.php?/cases/view/8517&group_by=cases:section_id&group_id=2161&group_order=asc&display_deleted_cases=0)
[some cerner](https://dsvavsp.testrail.io/index.php?/cases/view/13975&group_by=cases:section_id&group_id=2161&group_order=asc&display_deleted_cases=0)

**Comment by theodur** [comment]
I have no idea why the PR merge automatically closed the ticket, I didn't even see this ticket attached to the PR. Thanks for reopening Therese

**Comment by TKDickson** [comment]
RC script is fully updated with HSP cases; as that was the last QA-related post-develop-merge work, closing this ticket.
